tox.ini

```
@@ -1,8 +1,9 @@
 [tox]
 envlist = py3, pypy3, doc, pep8
+isolated_build = True
 
 [testenv]
-deps=-r test-requirements.txt
+deps=importlib-metadata
```

Can reuse the `dev` extras, also makes use of the conditional for `importlib-metadata`

```suggestion
extras=dev
```

I'm not sure if `tox` being in the extra will be a problem, is so, could have the `dev` extra with `tox` and a `test` extra with `importlib-metadata`

Or just remove `tox` from the `dev` extra